MediaBrowser.Providers/Plugins/Tmdb/TV/TmdbEpisodeProvider.cs

```diff
@@ -102,9 +104,48 @@ public async Task<MetadataResult<Episode>> GetMetadata(EpisodeInfo info, Cancell
                 return metadataResult;
             }
 
-            var episodeResult = await _tmdbClientManager
-                .GetEpisodeAsync(seriesTmdbId, seasonNumber.Value, episodeNumber.Value, info.SeriesDisplayOrder, info.MetadataLanguage, TmdbUtils.GetImageLanguagesParam(info.MetadataLanguage), cancellationToken)
-                .ConfigureAwait(false);
+            var episodeResult = new TvEpisode();
```

Comment:
This value never gets used?

Reply:
Required for this section of code to work

https://github.com/scampower3/jellyfin/blob/477d74d5c7be3c42bd5859134885d0e5afe3af30/MediaBrowser.Providers/Plugins/Tmdb/TV/TmdbEpisodeProvider.cs#L155C1-L256C40